app/src/sharedTest/java/org/simple/clinic/TestData.kt

```diff
@@ -987,13 +987,15 @@ object TestData {
       gender: Gender = randomGender(),
       dateOfBirth: LocalDate? = null,
       age: Age? = null,
+      patientCreatedAt: Instant = Instant.now(),
```

Comment:
Can we use a hardcoded `Instant` instead of the current timestamp? There should be a live template in the Simple tooling plugin that adds this.

Reply:
Going forward to all implementation in `TestData` will use `Instant.parse` right?

Reply:
Yes. Same for `LocalDate` as well.

Reply:
Done